shared/constants/network.ts

```diff
@@ -1263,6 +1281,9 @@ export const ETHERSCAN_SUPPORTED_NETWORKS = {
 export const CHAIN_ID_TO_GAS_LIMIT_BUFFER_MAP = {
   [CHAIN_IDS.OPTIMISM]: 1,
   [CHAIN_IDS.OPTIMISM_TESTNET]: 1,
+  [CHAIN_IDS.ACALA]: 1,
+  [CHAIN_IDS.ACALA_TESTNET]: 1,
+  [CHAIN_IDS.KARURA]: 1,
```

Comment:
@zjb0807 I just tested it myself, and I'm not sure these lines are necessary.  But I saw you wrote a little about manual testing and added screenshots.  Can you show me a very specific example that makes these lines necessary?

Reply:
We have encoded the storage fee data in `gasLimit`. If it is changed, the transaction will fail. We have increased the `gasLimit` before encoding, so the transaction will not fail with the returned gas.